src/test/java/resourceTests/PairResourceTests/PairRecourceFindPairsForDifferentAvailabilityTypes.java

```diff
@@ -0,0 +1,93 @@
+package resourceTests.PairResourceTests;
+
+import dao.SlotsDao;
+import domain.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import resources.PairResource;
+
+import java.sql.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+
+public class PairRecourceFindPairsForDifferentAvailabilityTypes {
+
+    private final Boolean isDev = true;
+    private final Boolean isTest = false;
+    private final Boolean isOps = false;
+    private String startDate;
+    private String endDate;
+
+    private PairResource resource;
+    private List<Slots> mockSlots = new ArrayList<>();
+
+    @Mock
+    private static SlotsDao mockDao;
+
+    @Before
+    public void setUp(){
+        Calendar calendar = Calendar.getInstance();
+        Date date = new Date(calendar.getTimeInMillis());
+        calendar.add(Calendar.DAY_OF_MONTH, 1);
+        Date nextDate = new Date(calendar.getTimeInMillis());
+
+        DateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy");
+        startDate = dateFormat.format(date);
+        endDate = dateFormat.format(nextDate);
+
+        AvailabilityTypes typeAvailable = MockDataUtil.createAvailableType((long) 1, AvailabilityTypesEnum.available);
+        AvailabilityTypes typeMaybe = MockDataUtil.createAvailableType((long) 5, AvailabilityTypesEnum.maybe);
+
```

Comment:
Is it possible to place test data in another file? This data seems similar in all test files.